clearwater-infrastructure/usr/share/clearwater/bin/set_ntp_ip

```diff
@@ -0,0 +1,67 @@
+#Copyright (C) 2016 Metaswitch Networks Ltd
```

Comment:
You want the full open-source copyright statement here (e.g. https://github.com/Metaswitch/clearwater-infrastructure/blob/master/clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached#L5). You probably also want `#!/usr/bin/python`, and I think that will make Github's syntax highlighting work.